german/schwimmen/insel/fs_innen/service/service.md

```diff
@@ -1,7 +1,9 @@
-Ein heller Gong ertönt, und eine freundliche Frauenstimme meldet sich aus dem Lautsprecher und fragt, was sie 
-für dich tun kann. Du bist im ersten Moment etwas überrascht, und überlegst, was du eigentlich wollen solltest. 
+Ein heller Gong ertönt, und eine freundliche Frauenstimme meldet sich aus dem Lautsprecher und fragt, was sie
+für dich tun kann. Du bist im ersten Moment etwas überrascht, und überlegst, was du eigentlich wollen solltest.
 
 Dann antwortest du: "Ich hätte eigentlich ganz gern ein kühles Altbier!".
-Im nächsten Moment öffnet sich mit einem leisen, metallischen Knackgeräusch eine kleine Luke in der rückseitigen 
-Fahrstuhlwand. Diese war dir noch gar nicht aufgefallen. Hinter der Luke befindet sich eine kleine Aussparung, 
+Im nächsten Moment öffnet sich mit einem leisen, metallischen Knackgeräusch eine kleine Luke in der rückseitigen
+Fahrstuhlwand. Diese war dir noch gar nicht aufgefallen. Hinter der Luke befindet sich eine kleine Aussparung,
 und dort steht auf einem Tablett tatsächlich ein schönes, offensichtlich gekühltes Bier.
+
+[Du trinkst das Bier und widmest dich wieder den schwarzen Druckknöpfen.](./fs_innen/fs_innen.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.